apple/OmnivoreKit/Sources/App/Views/Home/HomeFeedViewModel.swift

```diff
@@ -159,17 +163,33 @@ import Views
       cursor = queryResult.cursor
       if let username = dataService.currentViewer?.username {
         await dataService.prefetchPages(itemIDs: newItems.map(\.unwrappedID), username: username)
-        // Only preload the first item in the list. We are doing this during the beta
-        // because it will kick off the user's future items being automatically transcribed.
-        // This happens because when an article is saved, we check if the user has a recent
-        // listen. If they do, we will automatically transcribe their message.
-        if let first = newItems.filter({ !$0.isPDF }).first?.id {
-          _ = await audioController.preload(itemIDs: [first])
-        }
       }
     } else {
       updateFetchController(dataService: dataService)
     }
+  }
+
+  func loadItems(dataService: DataService, audioController _: AudioController, isRefresh: Bool) async {
+    let syncStartTime = Date()
+    let start = CFAbsoluteTimeGetCurrent()
+
+    isLoading = true
+    showLoadingBar = true
+
+    _ = await [
```

Comment:
Interesting syntax! Do you know if this loads concurrently?

Reply:
I believe so, but I think a taskGroup is a bit more clear. I'll switch to that